### DISCUSSION TITLE
How to create frames

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#198] [Q&A] [answered]
Hi, what are the details for the frame .png e.g. size.
What tool did you use to bild the default frames.

**Comment by mgineer85** [comment, marked as answer]
Hi, I used Inkscape. It needs to have a transparent area where the capture shall be mounted to. So the size needs to be as large as you need your resolution of the final images. The bigger the longer processing times.

**Reply by Striker62** [reply]
ok got you will try. If I can manage it is there a place to exchange frames with other user?

**Reply by mgineer85** [reply]
There is no such place. I assume it could be a problem due to copyright issues (at least for some frames)

**Comment by Striker62** [comment]
go you're right, didn't have this in mind
